public/index.html

```diff
@@ -43,24 +39,105 @@ <h5 id="campaignType">Campaign Type</h5>
     <div id="missionList"></div>
   </div>
 
+  <div id="dataDiv">test</div>
+
   <script>
     $.getJSON('/api/data', function(data) {
       $('#campaignName').text(data.name);
-      $('#campaignType').text(data.campaign_type);
+      $('#campaignType').text(data.campaignType);
 
       for (var index in data.completedMissions) {
         var mission = data.completedMissions[index];
         $('#missionList').append('<div class="completedMission">' + mission + '</div>');
       };
 
-      $('#imperialPlayer').text(data.imperialPlayer.name);
+      // Add infor for Imperial player
+      var imperialPlayer = $("<div/>").attr({
+        class:"well text-left",
+      });
+
+      // Create sub divs for each attribute
+      var imperialPlayerName = $("<h4/>").attr({
+        class:"playerName text-center"
+      }).text(data.imperialPlayer.name);
+
+      var classDeck = $("<div/>").attr({
+        class:"classDeck",
+      }).text("Class Deck: " + data.imperialPlayer.classDeck);
+
+      var totalXP = $("<div/>").attr({
+        class:"playerXP",
+      }).text("Total XP: " + data.imperialPlayer.totalXP);
 
+      var currentXP = $("<div/>").attr({
+        class:"playerXP",
+      }).text("Current XP: " + data.imperialPlayer.currentXP);
+
+      var influence = $("<div/>").attr({
+        class:"influence",
+      }).text("Influence: " + data.imperialPlayer.influence);
+
+      var upgrades = createListObject(data.imperialPlayer.upgrades, "upgrades", "Upgrades: ");
+
+      imperialPlayer.append(imperialPlayerName);
+      imperialPlayer.append(classDeck);
+      imperialPlayer.append(totalXP);
+      imperialPlayer.append(currentXP);
+      imperialPlayer.append(influence);
+      imperialPlayer.append(upgrades);
+      $('#imperialPlayer').append(imperialPlayer);
```

Comment:
This if fine, but you don't really want to append here right?  You want to replace all of the content in this div.  I think the .html() function does that but I'm not 100% certain.